src/js/module.js

```diff
@@ -275,7 +275,12 @@ Module.load = function(id, parent) {
     source = Builtin.readSource(modPath);
     module.exports = JSON.parse(source);
   } else if (dynamicloader && ext === 'node') {
-    module.exports = dynamicloader(modPath);
+    if (process.platform === 'windows') {
+      module.exports = dynamicloader(modPath.replace(new RegExp('/', 'g'),
```

Comment:
Why is this needed, windows uses the `/` for paths?

Reply:
[`LoadLibrary`](https://docs.microsoft.com/en-us/windows/desktop/api/libloaderapi/nf-libloaderapi-loadlibrarya) WINAPI function only works with windows paths and it it easier to replace the forward slashes in JS. Quote from the reference:

> When specifying a path, be sure to use backslashes (), not forward slashes (/). For more information about paths, see [Naming a File or Directory](https://msdn.microsoft.com/121cd5b2-e6fd-4eb4-99b4-b652d27b53e8).

Reply:
I see. then the RegExp should be only constructed once and not every time the require is called.

Reply:
You are right. I updated the `path` object with `toUnixPath` and `toWindowsPath`.